packages/wallets/src/components/WalletTourGuide/WalletTourGuideSettings.tsx

```diff
@@ -5,11 +5,193 @@ import { WalletButton, WalletText } from '../Base';
 import './WalletTourGuide.scss';
 
 export const tourStepConfig = (
+    isMobile: boolean,
     isDemoWallet: boolean,
     hasMT5Account: boolean,
     hasDerivAppsTradingAccount: boolean,
     isAllWalletsAlreadyAdded: boolean
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+): any[] =>
```

Comment:
Are these steps supposed to be of type any?

Reply:
@nijil-deriv We discussed it [here:](https://github.com/binary-com/deriv-app/pull/11312). Already have [PR](https://github.com/binary-com/deriv-app/pull/11312) to fix this